src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -45,7 +45,9 @@ function setClipboardMessage(content: string) {
     if (!Clipboard.canSetHtml()) {
         Clipboard.setString(parser.htmlToMarkdown(content));
     } else {
-        const plainText = parser.htmlToText(content);
+        const anchorRegex = /<a\s+(?:[^>]*?\s+)?href=['"](.*?)['"]/;
```

Comment:
Could we reuse this regex?

https://github.com/Expensify/App/blob/e8ae3c5acedf0e6788dc574c7b6f3043ca37092a/src/libs/ReportActionsUtils.ts#L403